src/App.css

```diff
@@ -17,7 +18,7 @@ html { font-family: 'Roboto', sans-serif; }
     border: 1px solid #DDDDDD;
     border-radius: 4px;
     box-shadow: #999999 2px 2px 2px;
-    margin: 8px;
+    margin: 20px 2%;
```

Comment:
Por que usa o 20px junto to 2% ?

20px é como um offsset? caso seja maior vira 2%

Abraço!